apps/aecore/src/aec_metrics.erl

```diff
@@ -34,6 +34,8 @@
 -record(st, {type_map = [] :: list(),
              prefix = []   :: string()}).
 
+-record(filter, {}).
```

Comment:
Would it not be more flexible and future proof to have a map here instead? Just an idea, not needed at the moment.

Reply:
Actually, it would be more consistent right now to make it an atom; it's only used for pattern-matching in order to determine plugin context, and the 'log' plugin uses an atom instead.